### ISSUE TITLE
UnicodeDecodeError when trying to load download page

### ISSUE THREAD

**Issue by johtso** [pyload/pyload#1265]
```
Traceback (most recent call last):
File "/usr/local/share/pyload/module/lib/bottle.py", line 733, in _handle
return route.call(**args)
File "/usr/local/share/pyload/module/lib/bottle.py", line 1448, in wrapper
rv = callback(*a, **ka)
File "/usr/local/share/pyload/module/web/utils.py", line 113, in _view
return func(*args, **kwargs)
File "/usr/local/share/pyload/module/web/pyload_app.py", line 209, in downloads
for file in sorted([fs_decode(x) for x in files]):
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 12: ordinal not in range(128)
```

There seems to be a lot of unicode issues in general. All my downloads that contain unicode characters fail.

**Comment by vuolter** [comment]
Fixed in 0.4.10

**Comment by Bruzzlee** [comment]
quick workaround (tested on kubuntu 16.04 / Python 2.7.12 ):
add following to /pyload/module/web/pyload_app.py after 
"from module.utils import formatSize, save_join, fs_encode, fs_decode"

reload(sys)
sys.setdefaultencoding('Cp1252')
